tasks/move-granules/index.js

```diff
@@ -232,26 +213,26 @@ async function moveGranuleFiles(granulesObject, sourceBucket) {
 * @returns {Promise} promise resolves when all files have been updated
 **/
 async function updateCmrFileAccessURLs(cmrFiles, granulesObject, allFiles, distEndpoint) {
-  for (const cmrFile of cmrFiles) {
+  cmrFiles.forEach(async (cmrFile) => {
```

Comment:
This is not going to work because the code doesn't wait for the operations inside the forEach.

You can change `forEach` with `map` and then have an `await Promise.all()` to make sure everything in the loop is completed.

Reply:
changed